src/method.jl

```diff
@@ -52,7 +52,7 @@ Implementation of an augmented Lagrangian method. The following keyword paramete
 function percival(::Val{:equ}, nlp :: AbstractNLPModel; μ :: Real = eltype(nlp.meta.x0)(10.0),
             max_iter :: Int = 1000, max_time :: Real = 30.0, max_eval :: Int=100000,
             atol :: Real = 1e-8, rtol :: Real = 1e-8, ctol :: Real = 1e-8,
-            subsolver_logger :: AbstractLogger=NullLogger(),
+            subsolver_logger :: AbstractLogger=NullLogger(), inity = nothing,
```

Comment:
Is `inity` supposed to be a vector here?

Reply:
yes

Reply:
`inity` should be added to the docstring above.